nmpolicy/internal/resolver/filter.go

```diff
@@ -56,11 +56,11 @@ func isEqual(obtainedValue interface{}, desiredValue ast.Node) (bool, error) {
 func mapContainsValue(mapToFilter map[string]interface{}, filterKey string, expectedNode ast.Node) (interface{}, error) {
 	obtainedValue, ok := mapToFilter[filterKey]
 	if !ok {
-		return nil, filterError("cannot find key %s in %v", filterKey, obtainedValue)
+		return nil, fmt.Errorf("cannot find key %s in %v", filterKey, obtainedValue)
```

Comment:
The message in the test helped me to understand the error should be changed to -
`fmt.Errorf("cannot find key %s in %v", filterKey, mapToFilter)`

Reply:
Done